app/src/main/java/org/wikipedia/page/leadimages/LeadImagesHandler.java

```diff
@@ -44,6 +67,9 @@
     @NonNull private final PageHeaderView pageHeaderView;
 
     private int displayHeightDp;
+    private SuggestedEditsSummary sourceSummary, targetSummary;
```

Comment:
Since either of these fields can be `null`, they should be annotated as `@Nullable`, so that the IDE will give you hints that might help you catch possible NPE cases, of which I see at least one already (see below).